roles/ceilometer-control/meta/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 dependencies:
+  - role: openstack-meta
```

Comment:
You just need to add openstack-meta to ceilometer-common. This role already depends on ceilometer-common